python/tvm/rpc/server.py

```diff
@@ -119,6 +119,15 @@ def download_linked_module(file_name):
     return temp
 
 
+def _serve_loop(sock, load_library, work_path=None):
```

Comment:
can we keep the parameter name consistent, like use " conn" all or use " sock" all? 
Is there requirements let work_path be a optional parameter?

Reply:
The earlier code that I copied from used "sock" to refer to the socket and "conn" to refer to the resulting object from accept(). The current code uses "sock" everywhere, as in "socket object" but not the same object throughout. It sounds like you prefer that?
No, I'm not sure why work_path was optional.

Reply:
even though the earlier code, there is only 1 function _serve_loop, the type of its 1st parameter is socket, we can only use 1 name "sock" or "conn", there is 2 caller of the function _serve_loop, just as you said, 1 caller's argument is a original socket, and another caller's argument is a result object from accept(), actually the type of the result object from accept still is socket, so I use the name "sock", because it closer with the type.